.github/workflows/python-package-conda.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       shell: bash -leo pipefail {0}
       run: |
         python -m ipykernel install --user --name eschool2022
-        pytest --nbmake --nbmake-kernel=eschool2022 **/*ipynb
+        pytest --nbmake --nbmake-kernel=eschool2022 --ignore=julia --ignore=testing
```

Comment:
revert please, I have added the metadata to completely ignore the julia notebook

Reply:
you can add the metadata to the notebook:
```
{
  "cells": [ ... ],
  "metadata": {
    "kernelspec": { ... },
    "execution": {
      "allow_errors": true,
      "timeout": 300
    }
  }
}
```